docs/CONTRIBUTING.md

```
@@ -1,37 +1,105 @@
+
+- Write clear and concise PR titles and descriptions.
+
+- Add tests if you add new features.
+
+- Update documentation if needed.
+
+
+### 🧹 Code Style Guidelines
+- Python: Follow PEP8 and use Black (black .).
+- Frontend: Follow Tailwind CSS best practices.
+- Run pre-commit hooks locally before submitting PRs. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)